lib/gems/pending/util/object_storage/miq_swift_storage.rb

```diff
@@ -0,0 +1,159 @@
+require 'util/miq_object_storage'
+
+class MiqSwiftStorage < MiqObjectStorage
+  attr_reader :container_name
+
+  def self.uri_scheme
+    "swift".freeze
+  end
+
+  def self.new_with_opts(opts)
+    new(opts.slice(:uri, :username, :password))
+  end
+
+  def initialize(settings)
+    super(settings)
+
+    # NOTE: This line to be removed once manageiq-ui-class region change implemented.
```

Comment:
Is this the case here also?

Also typo, `s/manageiq-ui-class/manageiq-ui-classic/`